govalid_test.go

```diff
@@ -296,19 +296,19 @@ func TestValidateCustomRule(t *testing.T) {
 }
 
 func vioMustInclude(t *testing.T, val any, msgs ...string) {
```

Comment:
It's a test helper

So it must start with t.Helper


```suggestion
func vioMustInclude(t *testing.T, val any, msgs ...string) {
  t.Helper()
```